lib/LANraragi/Controller/Api/Category.pm

```diff
@@ -138,5 +139,61 @@ sub remove_from_category {
     }
 }
 
+sub get_highlight_category {
```

Comment:
```suggestion
sub get_highlighted_category {
```  
I think we should update all the names to refer to a "highlighted category", instead of a highlight. It's a little thing, but it makes it more explicit that is is a specific flag on top of a given category ID.